<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DAKSHIT-HP" failures="2" tests="2" name="Sign In Test" time="21.077" errors="0" timestamp="2025-05-15T17:24:44 IST">
  <testcase classname="ShoppingOnline.Tests.ValidateLoginTest" name="InvalidLogin" time="15.441">
    <failure type="java.lang.AssertionError" message="expected [Your password or incorrect] but found [Your password is incorrect]">
      <![CDATA[java.lang.AssertionError: expected [Your password or incorrect] but found [Your password is incorrect]
at ShoppingOnline.Tests.ValidateLoginTest.InvalidLogin(ValidateLoginTest.java:40)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- InvalidLogin -->
  <testcase classname="ShoppingOnline.Tests.ValidateLoginTest" name="InvalidLogin" time="21.063">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;com.aventstack.extentreports.ExtentTest.fail(java.lang.Throwable)&amp;quot; because the return value of &amp;quot;java.lang.ThreadLocal.get()&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentTest.fail(java.lang.Throwable)" because the return value of "java.lang.ThreadLocal.get()" is null
at ShoppingOnline.TestComponents.Listeners.onTestFailure(Listeners.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- InvalidLogin -->
</testsuite> <!-- Sign In Test -->
